Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+# Cheatsheet
+## Commonly used functions on [Array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays)
+### [1. map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

```

Comment:
Please remove the numbering from these - it will make the hashlinks break more easily

Reply:
Done.